packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -59,17 +59,32 @@ export class AddJob {
       job.type === StepTypeEnum.DIGEST
         ? await this.addDigestJob.execute(AddDigestJobCommand.create({ job }))
         : undefined;
-    Logger.debug(`DigestAmount is: ${digestAmount}`, LOG_CONTEXT);
+
+    if (job.type === StepTypeEnum.DIGEST) {
+      Logger.debug(`DigestAmount is: ${digestAmount}`, LOG_CONTEXT);
+    }
+
+    if (job.type === StepTypeEnum.DIGEST && digestAmount === undefined) {
+      Logger.warn(
+        `Digest Amount does not exist on a digest job ${job._id}`,
+        LOG_CONTEXT
+      );
+
+      return;
+    }
```

Comment:
Amazing! had a lot of duplicate logs that are not needed

Reply:
This PR started because of them 😅